test does not compiles on older platforms after removing GTestColor

When I try to compile tests on debian stretch or buster I get the following error:

```
[ 38%] Building CXX object test/CMakeFiles/unit_tests.dir/error/error_test.cpp.o
In file included from /home/jkotan/sources/h5cpp/test/error/error_test.cpp:28:0:
/home/jkotan/sources/h5cpp/test/error/../gtest_print.hpp:34:27: error: variable or field ‘ColoredPrintf’ declared void
 extern void ColoredPrintf(GTestColor color, const char* fmt, ...);
                           ^~~~~~~~~~
/home/jkotan/sources/h5cpp/test/error/../gtest_print.hpp:34:27: error: ‘GTestColor’ was not declared in this scope
/home/jkotan/sources/h5cpp/test/error/../gtest_print.hpp:34:45: error: expected primary-expression before ‘const’
 extern void ColoredPrintf(GTestColor color, const char* fmt, ...);
                                             ^~~~~
/home/jkotan/sources/h5cpp/test/error/../gtest_print.hpp:34:62: error: expected primary-expression before ‘...’ token
 extern void ColoredPrintf(GTestColor color, const char* fmt, ...);
                                                              ^~~
/home/jkotan/sources/h5cpp/test/error/../gtest_print.hpp: In destructor ‘virtual TestCout::~TestCout()’:
/home/jkotan/sources/h5cpp/test/error/../gtest_print.hpp:37:27: error: ‘ColoredPrintf’ is not a member of ‘testing::internal’
 #define PRINTF(...)  do { testing::internal::ColoredPrintf(testing::internal::COLOR_GREEN, "[          ] "); testing::internal::ColoredPrintf(testing::internal::COLOR_YELLOW, __VA_ARGS__); } while(0)
                           ^
/home/jkotan/sources/h5cpp/test/error/../gtest_print.hpp:45:5: note: in expansion of macro ‘PRINTF’
     PRINTF("%s\n",str().c_str());
     ^~~~~~
/home/jkotan/sources/h5cpp/test/error/../gtest_print.hpp:37:60: error: ‘COLOR_GREEN’ is not a member of ‘testing::internal’
 #define PRINTF(...)  do { testing::internal::ColoredPrintf(testing::internal::COLOR_GREEN, "[          ] "); testing::internal::ColoredPrintf(testing::internal::COLOR_YELLOW, __VA_ARGS__); } while(0)
                                                            ^
/home/jkotan/sources/h5cpp/test/error/../gtest_print.hpp:45:5: note: in expansion of macro ‘PRINTF’
     PRINTF("%s\n",str().c_str());
     ^~~~~~
/home/jkotan/sources/h5cpp/test/error/../gtest_print.hpp:37:110: error: ‘ColoredPrintf’ is not a member of ‘testing::internal’
 #define PRINTF(...)  do { testing::internal::ColoredPrintf(testing::internal::COLOR_GREEN, "[          ] "); testing::internal::ColoredPrintf(testing::internal::COLOR_YELLOW, __VA_ARGS__); } while(0)
                                                                                                              ^
/home/jkotan/sources/h5cpp/test/error/../gtest_print.hpp:45:5: note: in expansion of macro ‘PRINTF’
     PRINTF("%s\n",str().c_str());
     ^~~~~~
In file included from /home/jkotan/sources/h5cpp/test/error/error_test.cpp:28:0:
/home/jkotan/sources/h5cpp/test/error/../gtest_print.hpp:37:143: error: ‘COLOR_YELLOW’ is not a member of ‘testing::internal’
 #define PRINTF(...)  do { testing::internal::ColoredPrintf(testing::internal::COLOR_GREEN, "[          ] "); testing::internal::ColoredPrintf(testing::internal::COLOR_YELLOW, __VA_ARGS__); } while(0)
                                                                                                                                               ^
/home/jkotan/sources/h5cpp/test/error/../gtest_print.hpp:45:5: note: in expansion of macro ‘PRINTF’
     PRINTF("%s\n",str().c_str());
     ^~~~~~
test/CMakeFiles/unit_tests.dir/build.make:114: recipe for target 'test/CMakeFiles/unit_tests.dir/error/error_test.cpp.o' failed
make[3]: *** [test/CMakeFiles/unit_tests.dir/error/error_test.cpp.o] Error 1
CMakeFiles/Makefile2:923: recipe for target 'test/CMakeFiles/unit_tests.dir/all' failed
make[2]: *** [test/CMakeFiles/unit_tests.dir/all] Error 2
CMakeFiles/Makefile2:957: recipe for target 'test/CMakeFiles/run_tests.dir/rule' failed
make[1]: *** [test/CMakeFiles/run_tests.dir/rule] Error 2
Makefile:255: recipe for target 'run_tests' failed
make: *** [run_tests] Error 2
```

--- a/test/gtest_print.hpp
+++ b/test/gtest_print.hpp
@@ -31,12 +31,14 @@ namespace testing
 {
 namespace internal
 {
+#ifndef GTestColor
 enum GTestColor {
   kDefault,
   kRed,
   kGreen,
   kYellow
 };
+#endif
 extern void ColoredPrintf(GTestColor color, const char* fmt, ...);
 }
 }

--- a/test/gtest_print.hpp
+++ b/test/gtest_print.hpp
@@ -31,18 +31,16 @@ namespace testing
 {
 namespace internal
 {
-#ifndef GTestColor
-enum GTestColor {
+enum LocalGTestColor {
   kDefault,
   kRed,
   kGreen,
   kYellow
 };
-#endif
-extern void ColoredPrintf(GTestColor color, const char* fmt, ...);
+extern void ColoredPrintf(LocalGTestColor color, const char* fmt, ...);
 }
 }
-#define PRINTF(...)  do { testing::internal::ColoredPrintf(testing::internal::kGreen, "[          ] "); testing::internal::ColoredPrintf(testing::internal::kYellow, __VA_ARGS__); } while(0)
+#define PRINTF(...)  do { testing::internal::ColoredPrintf(testing::internal::LocalGTestColor::kGreen, "[          ] "); testing::internal::ColoredPrintf(testing::internal::LocalGTestColor::kYellow, __VA_ARGS__); } while(0)
 
 // C++ stream interface
 class TestCout : public std::stringstream

--- a/test/gtest_print.hpp
+++ b/test/gtest_print.hpp
@@ -31,16 +31,16 @@ namespace testing
 {
 namespace internal
 {
-enum LocalGTestColor {
-  COLOR_DEFAULT,
-  COLOR_RED,
-  COLOR_GREEN,
-  COLOR_YELLOW
+enum GTestColor {
+  kDefault,
+  kRed,
+  kGreen,
+  kYellow
 };
-extern void ColoredPrintf(LocalGTestColor color, const char* fmt, ...);
+extern void ColoredPrintf(GTestColor color, const char* fmt, ...);
 }
 }
-#define PRINTF(...)  do { testing::internal::ColoredPrintf(testing::internal::COLOR_GREEN, "[          ] "); testing::internal::ColoredPrintf(testing::internal::COLOR_YELLOW, __VA_ARGS__); } while(0)
+#define PRINTF(...)  do { testing::internal::ColoredPrintf(testing::internal::kGreen, "[          ] "); testing::internal::ColoredPrintf(testing::internal::kYellow, __VA_ARGS__); } while(0)
 
 // C++ stream interface
 class TestCout : public std::stringstream

--- a/test/gtest_print.hpp
+++ b/test/gtest_print.hpp
@@ -31,7 +31,13 @@ namespace testing
 {
 namespace internal
 {
-extern void ColoredPrintf(GTestColor color, const char* fmt, ...);
+enum LocalGTestColor {
+  COLOR_DEFAULT,
+  COLOR_RED,
+  COLOR_GREEN,
+  COLOR_YELLOW
+};
+extern void ColoredPrintf(LocalGTestColor color, const char* fmt, ...);
 }
 }
 #define PRINTF(...)  do { testing::internal::ColoredPrintf(testing::internal::COLOR_GREEN, "[          ] "); testing::internal::ColoredPrintf(testing::internal::COLOR_YELLOW, __VA_ARGS__); } while(0)

